src/Storage/Query/Handler/SelectQueryHandler.php

```diff
@@ -15,7 +16,7 @@ class SelectQueryHandler
     /**
      * @param ContentQueryParser $contentQuery
      *
-     * @return QueryResultset
+     * @return QueryResultset|Content|null
```

Comment:
it's return `QueryResultset` or `Content`  or `null`. Find better, but don't set one type. Maybe mixed?